deploy/003-deploy-pools.ts

```diff
@@ -121,6 +121,11 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     tokenImplementation_: tokenImplementation.address,
   });
   await tx.wait();
+
+  comptroller1Proxy._setMarketBorrowCaps(
+    [tokenImplementation.address],
+    ["0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"],
+  );
```

Comment:
Shouldn't we set supply caps as well somewhere?

Reply:
The supply cap isn't using 0 for infinite supplies. The supply cap uses 0 as minting not allowed.

The borrow cap works differently because it skips code if the cap is infinite. Whereas supply always checks the cap

Reply:
Yeah, I mean this line allows to borrow tokens, but I can't find a line that sets supply cap from zero to 0xfff...fff. This means the supply should be impossible, because, as you said, 0 means minting not allowed 🤔